addons/website/views/snippets.xml

```diff
@@ -365,6 +365,74 @@
     </section>
 </template>
 
+<template id="s_company_team" name="Company Team">
+    <section class="s_company_team">
+        <div class="container">
+            <div class="row mb32">
+                <div class="col-md-12 col-xs-12">
+                    <h1 class="text-center">
+                        Meet the Executive Team
+                    </h1>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-6 col-xs-12 mb32">
+                    <div class="col-md-4">
```

Comment:
Actually it's a tweak to select image and text block together ( like this : https://goo.gl/gzrVhO ) so that it's convenient for end user to delete / duplicate the whole block while keeping it responsive.